plugins/postgresql/postgresql_test.go

```diff
@@ -12,7 +13,7 @@ func TestPostgresqlGeneratesMetrics(t *testing.T) {
 	p := &Postgresql{
 		Servers: []*Server{
 			{
-				Address:   "sslmode=disable",
+				Address:   fmt.Sprintf("host=%s user=postgres sslmode=disable", testutil.GetLocalHost()),
```

Comment:
This and the changes below it mean that the tests won't run as the current user any longer, which breaks the non-docker test setup.